homeassistant/components/input_boolean/__init__.py

```diff
@@ -52,6 +55,35 @@ def is_on(hass, entity_id):
     """Set up an input boolean."""
     component = EntityComponent(_LOGGER, DOMAIN, hass)
 
+    entities = await _async_process_config(config)
+    if not entities:
+        return False
```

Comment:
Let's not return false. It is totally fine to set up the integration without entities. We can reload and add more later !

Reply:
I would just not call `component.async_add_entities`, still register service.

Reply:
addressed in a31b0eb391bbb7222c9eb9ffaa931d30484908b3